libnavigation-core/src/test/java/com/mapbox/navigation/core/lifecycle/CarAppLifecycleOwnerTest.kt

```diff
@@ -141,8 +142,31 @@ class CarAppLifecycleOwnerTest {
             onResume(carAppLifecycleOwner)
             onPause(carAppLifecycleOwner)
             onStop(carAppLifecycleOwner)
-            onDestroy(carAppLifecycleOwner)
         }
+        carAppLifecycleOwner.activityLifecycleCallbacks.onActivityStopped(activity)
```

Comment:
```suggestion
        carAppLifecycleOwner.activityLifecycleCallbacks.onActivityPaused(activity)
        carAppLifecycleOwner.activityLifecycleCallbacks.onActivityStopped(activity)
```

This may not impact the test, but `onActivityResumed` is called so it seems Pause should be called before Stopped